association/association_analysis.py

```diff
@@ -36,6 +37,18 @@ def clean(df):
     df.drop(to_drop, inplace=True)
 
 
+def fill(pivot_table, mx):
```

Comment:
You might want to:

- Rename method to `association_matrix`
- Avoid using `mx` as input parameter
- Create a data-frame `mx` in the method body
- Returns `mx` as result